Octokit.Tests/Clients/StarredClientTests.cs

```diff
@@ -346,19 +345,31 @@ public class TheGetAllForUserMethod
         public class TheGetAllStargazersForRepoMethod
         {
             [Fact]
-            public void RequestsCorrectUrl()
+            public async Task RequestsCorrectUrl()
```

Comment:
Actually it looks like there is inconsistency in the code base with both `async` and regular examples... but im interested whether you were trying to standardise to always being `async` or whether you actually had a failing test here?